tapgarden/custodian.go

```diff
@@ -465,7 +561,26 @@ func (c *Custodian) watchInboundAssets() {
 		var err error
 		select {
 		case newAddr := <-c.addrSubscription.NewItemCreated.ChanOut():
-			err = c.importAddrToWallet(ctxStream, newAddr)
+			importErr := c.importAddrToWallet(ctxStream, newAddr)
+			switch {
```

Comment:
As there are only two branches, this one might be better as an if/else block.

Reply:
I’m using switch mainly for readability. Just flatter and easier for me to scan. I think the expressionless switch pattern is idiomatic Go and used fairly often in the standard library. But I could be wrong.